src/Client/OAuth2Client.php

```diff
@@ -82,7 +83,7 @@ public function redirect(array $scopes = [], array $options = [])
     /**
      * Call this after the user is redirected back to get the access token.
      *
-     * @return \League\OAuth2\Client\Token\AccessToken
+     * @return \League\OAuth2\Client\Token\AccessTokenInterface
```

Comment:
It won't help phpstan (it may read through it and see it's returning a AccessTokenInterface), but it may make what is happening a tad clearer for the rest of us.